src/Schema/AbstractSchemaManager.php

```diff
@@ -671,8 +671,7 @@ protected function _getPortableTableColumnList(string $table, string $database,
         foreach ($tableColumns as $tableColumn) {
             $column = $this->_getPortableTableColumnDefinition($tableColumn);
 
-            $name        = strtolower($column->getQuotedName($this->platform));
-            $list[$name] = $column;
+            $list[strtolower($column->getName())] = $column;
```

Comment:
Technically, it may be a breaking change but practically nobody will care.

The only reason why this might be useful is for Oracle to avoid conflicts between fully a upper-cased column name and non-fully upper-cased (e.g. `ID` and `id`). In this case, the latter will be enclosed in quotes, so even after lower-casing they will be different (`id` and `"id"`) respectively.

However, this is a half-measure because:
1. The behavior of quoting of introspected names is implemented only for Oracle but not for IBM DB2 or PostgreSQL which are also SQL-92 compliant and are prone to the same problem.
2. Even on Oracle, it won't work for columns like `Id` and `iD` – they both will be converted to `"id"` and collide.